### PR TITLE
bug 1524381 validating embedded etcd

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1524381

@reestr, @sdodson, FYI

@jianlinliu, PTAL

### CODE REVIEW

**Comment by jianlinliu** [comment]
@jiajliu PTAL.

File: upgrading/migrating_embedded_etcd.adoc [hunk 1]
```diff
@@ -34,6 +35,13 @@ This migration process performs the following steps:
 . Re-configure master to use the external etcd.
 . Start master.
 
+Additionally, the etcd API version since {product-title} 3.6 defaults to v3. Also, since
+{product-title} 3.7, the v3 is the only version allowed. Therefore, older
```
Comment (ahardin-rh) [inline upgrading/migrating_embedded_etcd.adoc]: s/the v3/v3

File: upgrading/migrating_embedded_etcd.adoc [hunk 2]
```diff
@@ -85,6 +76,30 @@ and list the host to migrate your etcd to:
...
 ----
++
+[IMPORTANT]
+====
+If you find `etcd` in the `[OSEv3:children]` section, and the `[etcd]` host
+group already contains host names, you do not need to migrate etcd. Do not 
+follow the remaining steps.
+====
+
+. Pull the latest subscription data from RHSM:
```
Comment (ahardin-rh) [inline upgrading/migrating_embedded_etcd.adoc]: s/RHSM/Red Hat Subscription Manager (RHSM)

**Comment by ahardin-rh** [comment]
@kalexand-rh Just a few minor comments from me. Otherwise, looks good!

**Comment by kalexand-rh** [comment]
Thank you @ahardin-rh!

**Pull request merged**